README.md

```diff
@@ -280,3 +307,72 @@ Here's a summary of the operating systems supported by Gruntwork as of January,
 * **Tests**: It's worth noting all of our tests currently run on Linux, which means Windows-specific bugs do slip through periodically. 
 
 * **Need Windows support?** The vast majority of our customers use Linux or Mac, so we haven't prioritized improving our Windows support. If you would like us to improve our Windows support, we would be happy to discuss a project to do so as part of our [Custom Module Development](https://gruntwork.io/custom-module-development/) process.
+
+
+### How to use the code
+
+There's a lot of great code here, but how do you use it in your own infracture and apps? There are two options:
+
+1. [Reference it from the Gruntwork repos](#reference-it-from-the-gruntwork-repos) (recommended)
+1. [Fork the code to your own repos](#fork-the-code-to-your-own-repos)
+
+Note, you must be a [Gruntwork Subscriber](https://www.gruntwork.io/pricing/) to use either of these options!
+
+#### Reference it from the Gruntwork repos
+
+The approach we recommend is to reference the code directly from the Gruntwork repos.
+
+**For Terraform code**, you can use a Gruntwork module by referencing the Git URL in the `source` parameter. For example, to use our [VPC module](https://github.com/gruntwork-io/module-vpc) in your code:
+
+```hcl
+# Create a VPC using the Gruntwork VPC module
+module "vpc_app" {
+  source = "git::git@github.com:gruntwork-io/module-vpc.git//modules/vpc-app?ref=v0.5.5"
+
+  vpc_name         = "prod"
+  aws_region       = "us-east-1"
+  cidr_block       = "10.0.0.0/18"
+  num_nat_gateways = 3
+}
+```
+
+A few things to note about the `source` URL above:
+
+1. **Versioned URL** (note the `ref=v0.5.5`): This gives you a known, fixed version of each module, with consistent and reproducible behavior. We use [semantic versioning](https://semver.org/)  for our code and publish release notes for each release so you know how to upgrade and if there were any backward incompatible changes. If you want to upgrade to new infrastructure, just bump the version number and run `terraform apply`!
+
+1. **SSH for authentication**: If you [associate an SSH key with your GitHub account](https://help.github.com/en/articles/adding-a-new-ssh-key-to-your-github-account) and use [ssh-agent](https://help.github.com/en/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#adding-your-ssh-key-to-the-ssh-agent), you will not be prompted for any passwords. You can use a similar approach with your machine users in CI / CD environments.
+
+**For binaries and scripts**, you can use the [gruntwork-installer](https://github.com/gruntwork-io/gruntwork-installer) with the `--repo` parameter pointing to Gruntwork repos. For example, to install the [ssh-grunt binary](https://github.com/gruntwork-io/module-security/tree/master/modules/ssh-grunt) (which lets you manage SSH access using an Identity Provider such as IAM, Google, ADFS, etc) on your EC2 Instances:
```

Comment:
Might want to mention that `gruntwork-installer` is a bash script, and thus won't work with Windows.

Reply:
Fixed, thanks! https://github.com/gruntwork-io/toc/commit/0723f2437cf1c4341c73ee91212713ba5096644a